contracts/modules/SwapsExternal/SwapsExternal.sol

```diff
@@ -116,17 +116,19 @@ contract SwapsExternal is State, VaultController, SwapsUser, PausableGuardian {
         address trader,
         address sourceToken,
         address destToken,
-        uint256 sourceTokenAmount,
-        bytes calldata payload)
+        uint256 tokenAmount,
+        bytes calldata payload,
+        bool isAmountIn)
```

Comment:
this is a breaking change probably talk to kevin. should we just add a new function instead ?

Reply:
if we do not set the current target to the 0 address it'll still exist.